src/gui/activitydata.h

```diff
@@ -69,6 +70,23 @@ class Activity
 
 
     Identifier ident() const;
+
+    void extractLink( const QString& s )
+    {
+        if (!s.isEmpty()) {
+            // if there is a real whitespace in the link, the part before the space
+            // is rendered as a link text
+            int charPos = s.lastIndexOf(QChar(' '));
+            if( charPos > -1 ) {
```

Comment:
(style:  we don't put spaces inside the parentheses, the space is before.  You could run git-clang-format, to fix all style issues.)